src/PortfolioSummary.test.js

```diff
@@ -1,7 +1,7 @@
 import * as Utilities from './utils/utilities';
 import * as DateUtils from '../utils/dateUtils';
 
-const today = DateUtils.getTodaysDateInIso().split('T')[0];
+const today = new Date().toISOString().split('T')[0];
```

Comment:
`const [today, _] = newDate().toISOString(.split('T');`
Also,  this gets you the date in UTC - is that what you want?

Reply:
yes

Reply:
- wait why the [today, _] ?? how is that better?